client/settings/digital-wallets/index.js

```diff
@@ -0,0 +1,132 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	CardDivider,
+	CheckboxControl,
+} from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+const DigitalWallets = () => {
+	const [ isEnabled, setIsEnabled ] = useState( false );
+	const [ sectionsStatus, setSectionsStatus ] = useState( {
+		checkout: true,
+		productPage: true,
+		cart: true,
+	} );
+
+	const makeHandleSectionStatusChange = ( section ) => ( status ) => {
+		setSectionsStatus( ( oldStatuses ) => ( {
+			...oldStatuses,
+			[ section ]: status,
+		} ) );
+	};
+
+	return (
+		<Card className="digital-wallets">
+			<CardBody size="large">
+				<CheckboxControl
+					checked={ isEnabled }
+					onChange={ setIsEnabled }
+					label={ __(
+						'Enable digital wallets & express payment methods',
+						'woocommerce-payments'
+					) }
+					/* eslint-disable jsx-a11y/anchor-has-content */
+					help={ interpolateComponents( {
+						mixedString: __(
+							// eslint-disable-next-line max-len
+							"By enabling this feature, you agree to {{stripeLink}}Stripe{{/stripeLink}}, {{appleLink}}Apple{{/appleLink}}, {{googleLink}}Google{{/googleLink}} and {{microsoftLink}}Microsoft{{/microsoftLink}}'s terms of use.",
+							'woocommerce-payments'
+						),
+						components: {
+							stripeLink: (
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://stripe.com/apple-pay/legal"
+								/>
+							),
+							appleLink: (
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://developer.apple.com/apple-pay/acceptable-use-guidelines-for-websites/"
+								/>
+							),
+							googleLink: (
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://androidpay.developers.google.com/terms/sellertos"
+								/>
+							),
+							microsoftLink: (
+								<a
+									target="_blank"
+									rel="noreferrer"
+									href="https://www.microsoft.com/en/servicesagreement/"
+								/>
+							),
+						},
+					} ) }
+					/* eslint-enable max-len */
```

Comment:
This is not exactly the rule that needs to be reenabled, this should be:

```suggestion
					/* eslint-enable jsx-a11y/anchor-has-content */
```

BTW this example might be an argument in favor of using the per-line approach. 😅

Reply:
🤦 my bad